types.go

```diff
@@ -112,3 +121,232 @@ func (msgs *Messages) GetTitleValue(placeholder string) string {
 	}
 	return s
 }
+
+func messagesToRlp(messages Messages) []interface{} {
+	values := make([]interface{}, 0)
```

Comment:
What if there are more messages here? Or is that not the case in v1.0.0?

Reply:
There are one `Messages` in Data and one Messages for each argument. This is only for 1.0.0.